decksite/main.py

```diff
@@ -252,6 +252,13 @@ def do_claim():
         return redirect(url_for('signup'))
     return retire(form)
 
+
+@APP.route('/rotationchanges')
```

Comment:
I think you're right that this wants to be part of a "season summary" or similar eventually but this will do for a first stab.